main.py

```diff
@@ -312,12 +314,14 @@ def test(
             same = np.argmax(target) == np.argmax(output)
             if same:
                 correct += 1
-            print("Label: {} Output: {} Correct: {}".format(target, output, same))
-        print(
-            "Accuracy = {} / {} = {} \nAvg Cost: {}".format(
-                correct, N, correct / N, np.average(costs)
+            if __debug__:
+                print("Label: {} Output: {} Correct: {}".format(target, output, same))
```

Comment:
```suggestion
                print(f"Label: {target} Output: {output} Correct: {same}")
```
Alternatively, just use multiple elements:
```python3
                print("Label:", target, "Output:", output, "Correct:", same)
```